test/index.spec.js

```diff
@@ -168,4 +168,17 @@ describe('ansible-vault', function () {
         })
     })
   })
+
+  describe('1.4', function () {
```

Comment:
can you please change the name "1.4" as being confusing to "sync operations" or similar? Currently there is no ansible vault in version 1.4.

Reply:
No problem. Done, Thanks.